i18n/qqq.json

```diff
@@ -48,13 +48,11 @@
 	"machinevision-cta-cta": "Call to action text for the button linking to the popular uploads tab",
 	"machinevision-no-uploads-cta-heading": "Heading for the user message when the user has not yet uploaded any images that have received tag suggestions",
 	"machinevision-no-uploads-cta-text": "Text for the user message when the user has not yet uploaded any images that have received tag suggestions",
-	"machinevision-generic-no-images-heading": "Heading for a generic message shown to user when there are no images to tag",
-	"machinevision-generic-no-images-text": "Text for a generic message shown to user when there are no images to tag",
```

Comment:
I noticed that these have been removed upstream (though we are using them in the fork here) – what should they be replaced with?